README.md

```diff
@@ -1,5 +1,11 @@
 [![AsyncAPI Generator](./assets/readme-banner.png)](https://www.asyncapi.com/tools/generator)
 
+This is a Monorepo managed using [Turborepo](https://turbo.build/) and contains the following package:
+
+1. [Generator-filters](apps/generator-filters):This is a library with generator filters that are used in more than one template. Use these filters to not double your work.
```

Comment:
less important than generator, please move as last on the list and indicate it is only for Nunjucks template and that it is released together with generator by default, that no need to add it to dependencies separately

Reply:
and yeah, make sure it is true :) so please add it to generator package.json